### ISSUE TITLE
Can not open message action menu using screen reader

### ISSUE THREAD

**Issue by dnz3d4c** [RocketChat/Rocket.Chat#12325]
### Description:
Can not open message action menu using screen reader

### Steps to reproduce:

1. Open any channel and navigate the message area to the screen reader.
2. Use a screen reader to see if there is a button to open the message action menu.

### Expected behavior:
You should be able to recognize and use buttons that can open message action menus using a screen reader.

### Actual behavior:

You can not recognize buttons that can open the message action menu using a screen reader, and you can not use them.
### Server Setup Information:
- Version of Rocket.Chat Server: rocketchat-server 0.69.2
- Operating System: ubuntu 18.04
- Deployment Method: 

Thanks.

**Comment by reetp** [comment]
Unfortunately this isn't really a 'bug' in the literal sense. It is an enhancement / NFR.

All NFRs should now be opened in the forums here:
https://forums.rocket.chat/c/feature-requests

I'll label it as enhancement and close it for now (albeit reluctantly as I think this should be a high priority enhancement for visually impaired users)

**Comment by reetp** [comment]
@rocket-cat label add enhancement

**Comment by reetp** [comment]
@rocket-cat close

**Comment by reetp** [comment]
This refers as well - please follow up there as well:
https://github.com/RocketChat/Rocket.Chat/issues/12061

**Comment by Sing-Li** [comment]
Thanks @reetp  -   @dnz3d4c please create a feature request on https://forums.rocket.chat/c/feature-requests    with title similar to "Support Screen Readers for Accessibility"  for wider community feedback  - thanks

**Comment by reetp** [comment]
@rocket-cat label add "Feature: Request"

**Comment by dnz3d4c** [comment]
@Sing-Li, @reetp I have written a feature request post: <https://forums.rocket.chat/t/improved-rocketchat-screen-reader-support/2290?u=dnz3d4c>
